docs/csharp/programming-guide/statements-expressions-operators/snippets/how-to-define-value-equality-for-a-type/ValueEqualityClass/Program.cs

```diff
@@ -0,0 +1,202 @@
+﻿namespace ValueEqualityClass
+{
+    using System;
+    class TwoDPoint : IEquatable<TwoDPoint>
+    {
+        // Readonly auto-implemented properties.
```

Comment:
I see that the code has been just moved without major updates, so I'm feeling free to suggest to remove all the obvious, won't-be-localized comments. Moreover, this comment is even wrong, as the properties are not readonly, as they can be modified not only by the constructor, but also by the type methods.

```suggestion
```